### ISSUE TITLE
Non intuitive UI on search Results Page

### ISSUE THREAD

**Issue by thesmallstar** [phpmyadmin/phpmyadmin#15961]
The UI is non-intuitive, links/buttons appear as texts.  We can replace those text with proper button to toggle results.
![image](https://user-images.githubusercontent.com/42006277/74585249-a06de480-5000-11ea-8d54-746a557304bf.png)

**Comment by thesmallstar** [comment]
Working on master branch :)

**Comment by OlafvdSpek** [comment]
It's not just styling that's non-intuitive.

Being able to hide results doesn't seem to make sense.

**Comment by thesmallstar** [comment]
YES, I would be more than happy to remove that. Should I proceed? @OlafvdSpek

**Comment by OlafvdSpek** [comment]
Yes please
